### DISCUSSION TITLE
Nightingale callback data

### DISCUSSION THREAD

**Discussion by LiangYang1018** [ccfos/nightingale#1161] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/69097273/191399025-7d813104-188b-4ff6-8331-37b30d275f8b.png)
启用恢复通知，当集群恢复后，回调地址也会收到回调吗？如果会的话，有没有恢复时回调数据格式的文档或样例呀？

**Comment by 710leo** [comment]
恢复通知也会触发回调，格式和告警用的是同一个数据结构

**Reply by LiangYang1018** [reply]
如果某条规则的第一次告警，没恢复的后续告警，以及恢复后的恢复通知都是调用同一个接口，那这些回调数据的哪一个字段标记它们属于同一条规则（告警事件）呢?   （是"hash"还是“rule_id”吗？）

**Reply by 710leo** [reply]
`hash` 字段

**Reply by LiangYang1018** [reply]
恢复通知哪一个字段表示恢复的时间呀？（"trigger_time"吗？）

**Comment by UlricQin** [comment, marked as answer]
`nc -l 4567` 用nc工具监听一个端口，然后把这个端口地址配置到回调地址那个框里，触发之后，n9e-server就会回调，这个nc就可以收到请求内容，你就可以在terminal里看到了，什么结构一目了然

**Reply by LiangYang1018** [reply]
有什么好的办法模拟告警恢复的场景吗？
我现在的想法是：在集群里找一个比较好调整的指标，来模拟恢复的场景。先触发了这个指标的告警，然后通过手动调整集群恢复了这个指标，从而触发了恢复的回调接口。
除了这种手动触发，还有更简单的触发恢复接口的方法吗？
